public/app/features/commandPalette/CommandPalette.tsx

```diff
@@ -65,7 +58,18 @@ export const CommandPalette = () => {
   useEffect(() => {
     if (showing) {
       reportInteraction('commandPalette_opened');
+
+      keybindingSrv.bindGlobal('esc', () => {
```

Comment:
One thing to note, we looked into the [Mousetrap pause/unpause extension](https://github.com/ccampbell/mousetrap/tree/master/plugins/pause), but that simply toggles the mousetrap bindings on and off. It does not prevent overrides, or resets overrides after being unpaused. An extension to mousetrap to set different keybinding profiles via a function call would be extremely useful!